pkg/products/threescale/reconciler.go

```diff
@@ -2896,9 +2921,7 @@ func (r *Reconciler) reconcileDeploymentConfigs(ctx context.Context, serverClien
 			ctx,
 			serverClient,
 			resources.SelectFromDeploymentConfig,
-			resources.AllMutationsOf(
-				resources.MutateZoneTopologySpreadConstraints("app"),
```

Comment:
is the entire `reconcileDeploymentConfigs` even required anymore? I'm not sure I'm missing something but it doesn't seem to be doing anything atm?

Reply:
Agree don't think its doing anything without the mutate , but like you I'm not sure I am missing something.

Reply:
removed `reconcileDeploymentConfigs`